pysages/methods/umbrella_integration.py

```diff
@@ -90,11 +90,13 @@ def run(  # pylint: disable=arguments-differ
             Passed to the backend run function as additional user arguments.
 
         * Note:
-            This method does not accepts a user defined callback are not available.
+            This method does not accept a user defined callback are not available.
```

Comment:
unclear to me what is meant here

Reply:
Usually all methods accept a callback and the user can write there whatever they want.
UmbrellaIntegration however, does not, because we need a callback for the analysis.

Reply:
And we currently haven't implemented anything that allows chaining of callbacks.

Reply:
```suggestion
            This method does not accept a user defined callback.
```

Reply:
Gotcha.